compiler/optimizer/LiveVariableInformation.cpp

```diff
@@ -229,7 +229,7 @@ void TR_LiveVariableInformation::initializeGenAndKillSetInfo(TR_BitVector **regu
          block = node->getBlock();
          blockNum = block->getNumber();
          seenException  = false;
-         if (_ignoreOSRUses && (block->isOSRCodeBlock() || block->isOSRCatchBlock()))
+         if (_ignoreOSRUses && (block->isOSRCodeBlock() || block->isOSRInduceBlock() || block->isOSRCatchBlock()))
```

Comment:
I don't completely understand this change. The original code was checking if OSR transition related uses could be ignored (based on the _ignoreOSRUses boolean) and I think this change seems to ignore blocks containing OSR induction call too. I don't see how this is legal because there could other uses that have nothing to do with OSR in a block ending in an OSR induction call.